package.json

```diff
@@ -29,7 +29,8 @@
     "test": "jake test",
     "build": "jake build",
     "release": "./build/publish.sh",
-    "rollup": "rollup src/Leaflet.js -f umd -n L > bundle.js"
+    "lint": "node_modules/.bin/eslint --fix src/",
```

Comment:
In my case, I put `rollup` and `jake` **manually** in my `$PATH` because I was tired of running them locally (but they have to be updated/upgraded separately). Not so with eslint.